tap-agent/src/agent/sender_account.rs

```diff
@@ -602,10 +678,16 @@ impl Actor for SenderAccount {
                     // if it's being tracked and we didn't receive any update from the non_final_last_ravs
                     // remove from the tracker
                     state.rav_tracker.update(*allocation_id, 0);
+                    PENDING_RAVS
+                        .with_label_values(&[&state.sender.to_string(), &allocation_id.to_string()])
+                        .set(0f64);
```

Comment:
Wouldn't it be more accurate to remove the value completely from the metrics, since we won't be tracking the actual value anymiore? Using `remove_label_values()`.

Reply:
Done